### PR TITLE
Added version information, corrected maximum size

### LINKED ISSUE #13311: What version of .NET Framework was gcAllowVeryLargeObjects introduced?
It would be nice to have a section that provides the minimal version of .NET Framework that this feature is required.

&gt; The maximum number of elements in an array is UInt32.MaxValue.

The max is Int32.MaxValue (not uint)

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 2e093d16-79e3-e4eb-323f-4e4ef7fe677b
* Version Independent ID: f673a7d5-3db0-3bb2-5077-a3186871fc71
* Content: [&lt;gcAllowVeryLargeObjects&gt; Element](https://docs.microsoft.com/en-us/dotnet/framework/configure-apps/file-schema/runtime/gcallowverylargeobjects-element)
* Content Source: [docs/framework/configure-apps/file-schema/runtime/gcallowverylargeobjects-element.md](https://github.com/dotnet/docs/blob/master/docs/framework/configure-apps/file-schema/runtime/gcallowverylargeobjects-element.md)
* Product: **dotnet-framework**
* GitHub Login: @rpetrusha
* Microsoft Alias: **ronpet**

### PR DESCRIPTION
**Pull request by rpetrusha**
## Added version information, corrected maximum size

Fixes #13311

@danmosemsft @Maoni0

### CODE REVIEW

File: docs/framework/configure-apps/file-schema/runtime/gcallowverylargeobjects-element.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ On 64-bit platforms, enables arrays that are greater than 2 gigabytes (GB) in to
 ## Remarks  
  Using this element in your application configuration file enables arrays that are larger than 2 GB in size, but does not change other limits on object size or array size:  
   
-- The maximum number of elements in an array is <xref:System.UInt32.MaxValue?displayProperty=nameWithType>.  
+- The maximum number of elements in an array is <xref:System.Int32.MaxValue?displayProperty=nameWithType>.
```
Comment (Maoni0) [inline docs/framework/configure-apps/file-schema/runtime/gcallowverylargeobjects-element.md]: why this change? it's declared as an unsigned int in the code...

**Comment by mairaw** [comment]
@rpetrusha can you look at @Maoni0's comment?

File: docs/framework/configure-apps/file-schema/runtime/gcallowverylargeobjects-element.md (hunk 1 above)
Comment (rpetrusha) [reply docs/framework/configure-apps/file-schema/runtime/gcallowverylargeobjects-element.md]: I've reverted the change, @Maoni0.

**Review by mairaw — APPROVED** [review]
